ext/handlers/filters/messages.go

```diff
@@ -32,6 +33,19 @@ func Text(msg *gotgbot.Message) bool {
 	return msg.Text != ""
 }
 
+func Regex(text string) Message {
+	return func(msg *gotgbot.Message) bool {
+		   if msg.Text == "" {
+				return false
+			}
+           res, err:= regexp.Match(text, []byte(msg.Text))
```

Comment:
You can improve performance by calling `r := regexp.Compile()` in the function body, and using `r.Match()` here instead.
This will avoid having to compile the same regex on every update. It will also allow you to return any errors raised in the `Compile`, which are currently swallowed with no details (making debugging difficult).

Also, I recommend using `MatchString` here, which will avoid the memory copy caused by the type conversion.